.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - '0.*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  release:
+  name: Create Release
+  runs-on: ubuntu-latest
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@master
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        body: |
+          Changes in this Release
+        draft: false
```

Comment:
Is it normal that the body contains only this string 'Changes in this Release' ?